src-tauri/Cargo.toml

```diff
@@ -11,22 +11,26 @@ rust-version = "1.70"
 tauri-build = { version = "1.0.3", features = [] }
 
 [dependencies]
-arklib = { git = "https://github.com/ARK-Builders/arklib", rev = "51cfa7d6" }
+arklib = { git = "https://github.com/ARK-Builders/arklib", rev = "5021266" }
```

Comment:
This commit refers to this PR:
- https://github.com/ARK-Builders/arklib/pull/69

It's fine for debugging, but seems that we should implement proper error handling before merging.

Reply:
@gwendalF could you check if it's difficult to implement error handling without `unwrap` in arklib?